Makefile

```diff
@@ -7,3 +7,9 @@ install:
 
 tests:
 	npm test;
+
+storybook:
+	git remote set-url origin "https://${GITHUB_TOKEN}@github.com/BBC-News/psammead.git"
```

Comment:
Note that if we move to the BBC org from BBC News(https://github.com/BBC-News/simorgh/issues/1063), we'll need to test the github token still works as expected & that this deployment works correctly.